components/scream/src/dynamics/homme/atmosphere_dynamics_fv_phys.cpp

```diff
@@ -230,9 +230,10 @@ static TraceGasesWorkaround s_tgw;
 void fv_phys_rrtmgp_active_gases_init (const ekat::ParameterList& p) {
   const auto& v = p.sublist("atmosphere_processes").sublist("physics")
     .sublist("rrtmgp").get<std::vector<std::string>>("active_gases");
-  for (const auto& e : v)
-    if (e != "h2o")
-      s_tgw.active_gases.push_back(e);
+  for (const auto& e : v) {
+    if (e == "o3")
```

Comment:
So h2o was not an active gas before (was it the same as qv?), but now it is? Just making sure this did not get lost in the fixes.

Reply:
This section is only for the gases that need to be read from file and are not handled elsewhere. h2o is just qv, for which the dyn<->phys mapping happens elsewhere, and the rest of the active gases are computed within the radiation interface, so now we only need this special workaround code for o3 now (this too will go away when we can do simple online remap and can use the o3 climatology file that EAM uses for prescribed ozone cases, or if/when we port the linear ozone scheme to EAMxx).

Reply:
So you could basically just remove the for loop and do something like
```
if (ekat::contains(v,"o3")) {
  s_tgw.active_gases.push_back( .. );
}
```

Reply:
Yes, replaced.